src/__tests__/realmRoles.test.ts

```diff
@@ -72,4 +72,23 @@ describe('Realm Roles', () => {
     expect(listed).toHaveLength(3)
     expect(listed.map(r => r.id)).not.toContain(chef.id)
   })
+
+  it('describes a role by id', async () => {
+    const chef = await client.createRealmRole(realmName, {
+      name: 'Chef',
+      description: 'They cook things.',
+    })
+
+    const described = await client.describeRealmRole(realmName, chef.id)
+    expect(described.id).toBe(chef.id)
+    expect(described.name).toBe('Chef')
+
+    // sad but true, we throw 500 on nonexistent entity, not 404
```

Comment:
i've confirmed this is the case for describing both groups and roles. the problem originates from keycloak only returning a nondescript 400 error that our server converts to a 500 (rightfully, because it can't know _why_ it was a 400). fixing would require changes to the keycloak client.